share/wake/lib/core/syntax.wake

```diff
@@ -97,6 +97,12 @@ export def wait (f: a => b) (x: a): b =
 # 200
 # ```
 export def unreachable (reason: String): a =
-    def f x = prim "unreachable"
+    def f x = prim "panic"
```

Comment:
Should this stay as `prim "unreachable"`, since the comment you wrote in the primitive function registration code says that we do want to differentiate between panic and unreachable?

Reply:
Yep! Good catch!